CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+
+### Improvment
+- Upgrade to Svelte 4
```

Comment:
Wouldn't a major version bump be a breaking change?

Reply:
Ok, I see you have stated 1 -> 2 below, therefore this is not an "Improvement", it is a "Change"

- [ ] Use "Changed" instead of "Improvement" because it is a breaking change and used a major version bump.

Reply:
changed to `Changed`